config/features.yml

```diff
@@ -119,10 +119,6 @@ features:
     actor_type: user
     description: Enables the After Visit Summary API.
     enable_in_development: true
-  bcas_letters_use_lighthouse:
-    actor_type: user
-    description: Use lighthouse instead of EVSS to view/download benefit letters.
-    enable_in_development: true
   benefits_documents_use_lighthouse:
```

Comment:
Once this is deployed, you'll need to follow step 4 of [this section of this guide](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#FeatureTogglesGuide-Removingold/unusedfeaturetoggles) to completely remove this toggle. You'll need to do it in the terminal of each ArgoCD environment. If you or no one on your team has access to those terminals, let me know and I can do it for you. Remember, don't delete them in Argo until the code has been deployed.